src/status_im/contexts/wallet/common/sheets/account_origin/style.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.contexts.wallet.common.sheets.account-origin.style)
```

Comment:
beyond the scope of your pr, but I think `sheets` should move up one level 🤔 

i.e `status-im.contexts.wallet.bottom-sheets...`